### PR TITLE
chore: set up CI on v1 branch

### PR DESCRIPTION
**Pull request by busunkim96**
Set up GitHub Actions, update release script on v1

### CODE REVIEW

**Comment by parthea** [comment]
Please see the commit [here](https://github.com/googleapis/python-containeranalysis/pull/260/commits/b7c33423c6269b948075e1651f7425445c7816d7) needed to fix the docs build

**Comment by busunkim96** [comment]
The GitHub Actions `docs` check is green.

**Comment by parthea** [comment]
Can we remove docfx check from the Kokoro check or fix it using the commit [here](https://github.com/googleapis/python-containeranalysis/pull/260/commits/b7c33423c6269b948075e1651f7425445c7816d7)?

File: noxfile.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def docs(session):
     """Build the docs for this library."""
 
     session.install("-e", ".")
-    session.install("sphinx<3.0.0", "alabaster", "recommonmark")
+    session.install("sphinx<3.0.0", "alabaster", "recommonmark", "Jinja2<3.1")
```
Comment (busunkim96) [inline noxfile.py:152]: See https://github.com/sphinx-doc/sphinx/issues/10291

**Review by parthea — APPROVED** [review]

**Pull request merged**